client/assets/stylesheets/_nav-unification.scss

```
@@ -144,6 +145,30 @@
 	}
 
 	@include breakpoint-deprecated( '<480px' ) {
+		// New post button
+		.masterbar__item-new {
+			background: #32373c;
+		}
+
+		.masterbar__toggle-drafts.button.is-borderless {
+			background: #32373c;
+			color: #eeeeee;
```

We could use an existing variable here as well:

```suggestion
			color: var( --color-sidebar-text );
```

Generally, I'd suggest trying to reuse variables as much as possible instead of hardcoding colors.